case-study.Rmd

```diff
@@ -0,0 +1,467 @@
+---
+title: "ADAGE signature analysis - case study"
+author: "Jie Tan"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    fig_width: 7
+    fig_height: 7
+vignette: >
+  %\VignetteIndexEntry{User input example}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+```
+
+This script shows how we analyze the *∆anr* mutation dataset used as a case study
+in the ADAGE signature analysis paper. 
+
+# Data preparation
+Load in required libraries.
+
+```{r}
+library("ADAGEpath")
+library("DT")
+library("knitr")
+library("VennDiagram")
+library("readr")
+```
+
+Before any analysis, we need to specify the ADAGE model and the data compendium
+we want to use.
+
+```{r}
+model <- eADAGEmodel
+compendium <- PAcompendium
+probe_dist <- probedistribution
+```
+
+Let's load in the sample dataset that come with the package. It's expression data
+from *Pseudomonas aeruginosa* wild type and *∆anr* grown as biofilms on ∆F508 cystic
+fibrosis bronchial epithelial cells (CFBEs). Detailed information about
+the dataset can be found here
+[GSE67006](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67006).
+All its CEL files are stored in the folder "./inst/extdata/anr".
+To load your own dataset, simply modify this input path.
+
+```{r}
+input_path <- system.file("extdata", "anr/", package = "ADAGEpath")
+data_raw <- load_dataset(input = input_path, isProcessed = FALSE,
+                         isRNAseq = FALSE, model = model,
+                         compendium = compendium, quantile_ref = probe_dist,
+                         norm01 = FALSE)
+```
+
+ADAGE only accepts expression values in the (0,1) range. We linearly transform
+expression values to be between 0 and 1 using the Pa compendium as the reference.
+
+```{r}
+data_normed <- zeroone_norm(input_data = data_raw, use_ref = TRUE,
```

Comment:
The majority of functions here are in `ADAGEpath`. I used `namespace` when using functions from other libraries but not `ADAGEpath`. I think it's too much.